### PR TITLE
added server logging for error code 10 [BA-5703]

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala [hunk 1]
```diff
@@ -602,7 +603,13 @@ class PipelinesApiAsyncBackendJobExecutionActor(override val standardParams: Sta
           && errorMsg.contains("Execution failed")
           && (errorMsg.contains("Localization") || errorMsg.contains("Delocalization"))) {
         s"Please check the log file for more details: $jesLogPath."
-      } else errorMsg
+      }
+      //If error code 10, add some extra messaging to the server logging
+      if (runStatus.errorCode.getCode.value == PapiErrorCode10) {
```
Comment (danbills) [inline supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala]: I think you want an `else if` so as not to throw away the above message

File: supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala [hunk 2]
```diff
@@ -66,6 +66,7 @@ object PipelinesApiAsyncBackendJobExecutionActor {
   val JesPreemption = 14
 
   val PapiFailedPreConditionErrorCode = 9
+  val PapiErrorCode10 = 10
```
Comment (kshakir) [inline supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala]: Can we name this something more descriptive, `PapiPreemptionErrorCode` perhaps?

Comment (aednichols) [reply supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala]: I think the whole problem is that 10 is supposed to mean "crashed" but is being used to mean "pre-empted"?

Comment (kshakir) [reply supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala]: Ok, `PapiCrashedErrorCode` perhaps?

Comment (ruchim) [reply supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala]: Sure! I can rename it as suggested

File: supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala (hunk 1 above)
Comment (ruchim) [reply supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala]: Yes!

File: supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala [hunk 3]
```diff
@@ -66,6 +66,7 @@ object PipelinesApiAsyncBackendJobExecutionActor {
   val JesPreemption = 14
 
   val PapiFailedPreConditionErrorCode = 9
+  val PapiMysteriouslyCrashedErrorCode = 10
```
Comment (aednichols) [inline supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala]: 💯

**Pull request merged**